src/utils.ts

```diff
@@ -353,23 +357,23 @@ export class NumberHolder {
   }
 }
 
-/** @deprecated Use {@linkcode NumberHolder} */
-export class IntegerHolder extends NumberHolder {
-  constructor(value: number) {
-    super(value);
-  }
-}
-
-/** @deprecated Use {@linkcode NumberHolder}*/
-export class FixedInt extends IntegerHolder {
+/**
+ * Holds a fixed number value, this is solely used to differentiate between a regular number and a fixed number.
+ * @see `transformValue` in {@linkcode initGameSpeed}
+ * @extends NumberHolder
+ */
+export class FixedNumber extends NumberHolder {
   constructor(value: number) {
     super(value);
   }
 }
```

Comment:
```suggestion
export class FixedNumber {
  public readonly value: number;

  constructor(value: number) {
    this.value = value;
  }
}
```
Based on flx's suggestion.